doc/plutus/tutorials/HelloWorldApp.hs

```diff
@@ -1,29 +0,0 @@
-{-# LANGUAGE DataKinds        #-}
-{-# LANGUAGE PolyKinds        #-}
-{-# LANGUAGE TemplateHaskell  #-}
-{-# LANGUAGE TypeApplications #-}
-{-# LANGUAGE TypeFamilies     #-}
-{-# LANGUAGE TypeOperators    #-}
-module HelloWorldApp where
-
-import Data.Text qualified as T
-import Playground.Contract (KnownCurrency, mkKnownCurrencies, mkSchemaDefinitions)
-import Plutus.Contract (Contract, EmptySchema, Endpoint, logInfo)
-import PlutusTx.Prelude ()
-
--- BLOCK1
-
--- | A 'Contract' that logs a message.
-hello :: Contract () EmptySchema T.Text ()
-hello = logInfo @String "Hello, world"
-
--- BLOCK2
-
-endpoints :: Contract () EmptySchema T.Text ()
-endpoints = hello
-
-type DummySchema = Endpoint "dummy" ()
-
-mkSchemaDefinitions ''DummySchema
-
-$(mkKnownCurrencies [])
```

Comment:
@ak3n because `mkSchemaDefinitions` and `mkKnownCurrencies` are deleted in this PR. I should actually keep BLOCK1. Should I also keep BLOCK2 as below? or not worth it?

```
-- BLOCK2

endpoints :: Contract () EmptySchema T.Text ()
endpoints = hello

type DummySchema = Endpoint "dummy" ()
```